common/js/src/popup-window/popup-opener.js

```diff
@@ -71,6 +71,13 @@ GSC.PopupOpener.createWindow = function(url, createWindowOptions, opt_data) {
       chrome.app.window.create(
           url, createWindowOptions,
           createWindowCallback.bind(null, createdWindowExtends));
+    } else if (GSC.Packaging.MODE === GSC.Packaging.Mode.EXTENSION){
+      chrome.windows.create({
+        url: url,
+        type: 'popup',
+        width: createWindowOptions['innerBounds']['width'],
+        height: createWindowOptions['innerBounds']['height']
```

Comment:
Do all callers actually provide width and height? E.g., user-prompting-checker.js only specifies the width (see the `USER_PROMPT_DIALOG_WINDOW_OPTIONS_OVERRIDES` constant). Or does it work because non-existing property evaluates to `undefined`, and `chrome.windows.create()` ignores properties whose values are undefined?

Another question is whether we even want to pass `height` anywhere? You already saw the code in `GSC.InPopupMainScript.setWindowHeightToFitContent()` - it automatically resizes the window to fit the content. If that's indeed working as intended, then perhaps no caller should provide the height? It'd be a good thing to try out.

Reply:
> Do all callers actually provide width and height?
some calls to GSC.PopupOpener.createWindow use both width and height, so it's needed

> Another question is whether we even want to pass height anywhere?
you're right, GSC.InPopupMainScript.setWindowHeightToFitContent() should be enough to handle the height and indeed with trying to remove/keep height property didn't cause any difference in the window appearance